proposals/lip-0063.md

```diff
@@ -299,13 +301,15 @@ Some new LIPs introduced configurable constants. The values for these constants
 | [PoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants) | `LOCKING_PERIOD_SELF_STAKING` | 260,000 |
 | [PoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants) | `PUNISHMENT_WINDOW_STAKING` | 260,000 |
 | [PoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants) | `PUNISHMENT_WINDOW_SELF_STAKING` | 780,000 |
+| [PoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants) | `REPORT_MISBEHAVIOR_REWARD` | 10^8 |
 | [PoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants) | `REPORT_MISBEHAVIOR_LIMIT_BANNED` | 5 |
-| [PoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants) | `ROUND_LENGTH` | 103 |
 | [PoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants) | `MIN_WEIGHT` | 1000 x (10^8)
 | [PoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants) | `NUMBER_ACTIVE_VALIDATORS` | 101 |
 | [PoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants) | `NUMBER_STANDBY_VALIDATORS` | 2 |
 | [PoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants) | `TOKEN_ID_POS` | `TOKEN_ID_LSK` |
 | [PoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants) | `VALIDATOR_REGISTRATION_FEE` | 10 x (10^8) |
+| [PoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants) | `WEIGHT_SCALE_FACTOR` | 1000 * (10)^8 |
+| [PoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants) | `MAX_BFT_WEIGHT_CAP ` | 1000 |
 | [PoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants) | `INVALID_BLS_KEYS_IN_GENESIS_BLOCK` | `True` |
 | [Random Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0046.md#constants-and-config-parameters) | `MAX_LENGTH_VALIDATOR_REVEALS` | 206 |
 | [Fee Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0048.md#notation-and-constants) | `MIN_FEE_PER_BYTE` | 1000 |
```

Comment:
Comment for fee module. It seems to me that two new config constants are introduced, `MAX_BLOCK_HEIGHT_ZERO_FEE_PER_BYTE` and `ADDRESS_FEE_POOL`. For mainchain they don't introduce any protocol change, but I guess should be defined. We could check with LIP champion and/or devs, but I think that this is the case.

Reply:
Thank. I added them to the table.